sabot/kernel/src/test/java/com/dremio/exec/physical/impl/TestNativeFunctions.java

```diff
@@ -297,6 +298,12 @@ public void testFromTimeStamp() throws Exception {
       {"extractMonth(c0)", ts("1970-01-02T10:20:33"), 1l},
       {"extractYear(c0)", ts("1970-01-02T10:20:33"), 1970l},
       {"extractSecond(c0)", ts("1970-01-02T10:20:33"), 33l},
+      {"minute(c0)", ts("1970-01-02T10:20:33"), 20l},
```

Comment:
are all the tests added in TestNativeFunctions merged in gandiva?

Reply:
I see some of them are in open PRs. Lets add testcases in TestNativeFunctions for only the gandiva functions in a separate PR. In this way the java and gandiva impls will be independent.

For the java functions, you can add tests in other classes like TestStringFunctions, TestFunctionsQuery, etc or create a new class